docs/4. repository/1. sourcify-database.mdx

```diff
@@ -0,0 +1,106 @@
+# Sourcify Database
+
+Sourcify Database is the main storage backend for Sourcify. It is a PostgreSQL database that follows the [Verified Alliance Schema](https://github.com/verifier-alliance/database-specs) as its base with few modifications.
+
+You can follow the [`services/database/migrations`](https://github.com/ethereum/sourcify/tree/staging/services/database/migrations) folder for the initial schema and the changes made to it.
```

Comment:
I think it's worth expanding this below with some information about the `sourcify_matches` table (like you did for `verified_contracts`)